fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/item/v1/FabricItemSettings.java

```diff
@@ -32,6 +32,42 @@
  * {@code new FabricItemSettings()}.
  */
 public class FabricItemSettings extends Item.Settings {
+	public enum AttackSound {
```

Comment:
I would call this `AttackSoundType`

Reply:
I was considering just making it SoundType so i could add BURP and some others, because they are in the same position